apps/router/src/modals/ConnectServiceModal.tsx

```diff
@@ -110,7 +110,7 @@ export const ConnectServiceModal: React.FC<ConnectServiceModalProps> = ({
           <FormControl isInvalid={!!error}>
             <FormLabel>{t('home.connect-service-modal.url-label')}</FormLabel>
             <Input
-              placeholder='wss://fedimintd.domain.com:6000'
+              placeholder='fedimintd or gatewayd url'
```

Comment:
nit: i like the example personally so people know what they are looking for, consider:

`placeholder='fedimintd / gatewayd URL (ex: wss://fedimintd.domain.com:6000)'`

Reply:
I remember when I first used this UI I felt that the placeholder suggested I could only enter websocket urls here. However, I've reverted back to the original value but open to suggestions from others too on what's best here.

Reply:
Ohh i guess I assumed this must be a websocket URL so if there are common cases where it's not then the example isn't as helpful. No string opinion here!